PythonScript/src/NotepadPlusWrapper.cpp

```diff
@@ -900,6 +917,162 @@ boost::python::object NotepadPlusWrapper::allocateMarker(int quantity)
 	}
 }
 
+LRESULT NotepadPlusWrapper::getMenuHandle(int menu = 0)
+{
+	return callNotepad(NPPM_GETMENUHANDLE, static_cast<WPARAM>(menu), 0);
+}
+
+bool NotepadPlusWrapper::isTabBarHidden()
+{
+	return 1 == callNotepad(NPPM_ISTABBARHIDDEN);
+}
+
+bool NotepadPlusWrapper::hideToolBar(bool hideOrNot)
+{
+	return callNotepad(NPPM_HIDETOOLBAR, 0, hideOrNot);
+}
+
+bool NotepadPlusWrapper::isToolBarHidden()
+{
+	return callNotepad(NPPM_ISTOOLBARHIDDEN);
+}
+
+bool NotepadPlusWrapper::hideMenu(bool hideOrNot)
+{
+	return callNotepad(NPPM_HIDEMENU, 0, hideOrNot);
+}
+
+bool NotepadPlusWrapper::isMenuHidden()
+{
+	return callNotepad(NPPM_ISMENUHIDDEN);
+}
+
+bool NotepadPlusWrapper::hideStatusBar(bool hideOrNot)
+{
+	return callNotepad(NPPM_HIDESTATUSBAR, 0, hideOrNot);
+}
+
+bool NotepadPlusWrapper::isStatusBarHidden()
+{
+	return callNotepad(NPPM_ISSTATUSBARHIDDEN);
+}
+
+void NotepadPlusWrapper::saveFile(boost::python::str filename)
+{
+	std::shared_ptr<TCHAR> s = WcharMbcsConverter::char2tchar((const char*)boost::python::extract<const char*>(filename));

```

Comment:
Is this behaving correctly with unicode file names?

Reply:
Yes, I can open, activate, reload, save etc... without a problem but
notepad.launchFindInFilesDlg seems to be behaving different.